json_lines/_lib.py

```diff
@@ -132,7 +132,7 @@ def _mark_good(self, file, buffered):
                                                           buffered)
 
 
-def _path_to_str(path):
+def path_to_str(path):
```

Comment:
this function looks less useful in general, do you have any specific use case in mind?

Reply:
Don't have any specific case in mind. I made it public because I believe it can be useful for all functions that don't support `Path`.
I can make it private again if you don't agree - I'm concerning that as nice feature to have rather than something really needed.

Reply:
ok, fair enough!